elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -465,6 +465,12 @@ defmodule API.Client.Channel do
           OpenTelemetry.Tracer.set_status(:error, "not_found")
           {:reply, {:error, %{reason: :not_found}}, socket}
 
+        {:error, {:forbidden, violated_properties: violated_properties}} ->
```

Comment:
@conectado this is the only change that connlib needs to support and pass down to clients.

Reply:
What is the schema of `violated_properties`?

Reply:
It's just list of strings